pkg/authorization/authorizer/authorizer.go

```diff
@@ -394,6 +452,27 @@ func (a *openshiftAuthorizationAttributeBuilder) GetAttributes(req *http.Request
 		return nil, errors.New("could not get user")
 	}
 
+	urlSegments := splitPath(req.URL.Path)
+	isNonResourceURL := (len(urlSegments) <= 1) || !a.infoResolver.APIPrefixes.Has(urlSegments[0])
```

Comment:
A comment with examples of what are considered non-resource urls might be helpful. This also assumes APIPrefixes are single-segment... that's probably an assumption upstream as well... just pointing it out